src/amo/components/Home/index.js

```diff
@@ -211,78 +251,74 @@ export class HomeBase extends React.Component {
           {this.renderCuratedCollections()}
         </Card>
 
-        <LandingAddonsCard
-          addonInstallSource={INSTALL_SOURCE_FEATURED}
-          addons={featuredExtensions}
-          className="Home-FeaturedExtensions"
-          header={i18n.gettext('Featured extensions')}
-          footerText={i18n.gettext('See more featured extensions')}
-          footerLink={{
-            pathname: '/search/',
-            query: {
-              addonType: ADDON_TYPE_EXTENSION,
-              featured: true,
-            },
-          }}
-          loading={loading}
-        />
+        {includeFeaturedThemes && (
+          <LandingAddonsCard
+            addonInstallSource={INSTALL_SOURCE_FEATURED}
+            addons={featuredThemes}
+            className="Home-FeaturedThemes"
+            header={i18n.gettext('Featured themes')}
+            footerText={i18n.gettext('See more featured themes')}
+            footerLink={{
+              pathname: '/search/',
+              query: {
+                addonType: getAddonTypeFilter(ADDON_TYPE_THEME, {
+                  _config: this.props._config,
+                }),
+                featured: true,
+              },
+            }}
+            loading={loading}
+          />
+        )}
 
         {(loading || collections[0]) && (
-          <LandingAddonsCard
-            addonInstallSource={INSTALL_SOURCE_FEATURED_COLLECTION}
+          <FeaturedCollectionCard
             addons={collections[0]}
             className="Home-FeaturedCollection"
-            header={i18n.gettext('Translation tools')}
-            footerText={i18n.gettext('See more translation tools')}
-            footerLink={`/collections/${FEATURED_COLLECTIONS[0].username}/${
-              FEATURED_COLLECTIONS[0].slug
-            }/`}
+            collectionMetadata={featuredCollectionsMetadata[0]}
```

Comment:
The pattern of passing in a complex object is not very React-y and I think it will make some things difficult. Flow validation of the property values is a bit cumbersome, for example.

Why not just pass them in as props? You could spread the object into props, which would be fine.

```js
<FeaturedCollectionCard
  addons={collections[0]}
  className="Home-FeaturedCollection"
  {...featuredCollectionsMetadata[0]}
/>
```

Using spreads without Flow coverage is a little risky but it's still OK.

Reply:
Yes, this looks better. I could add Flow coverage to this file, but I guess I've already done way more than was required for issue #5501 ;-)